src/EditorFeatures/Test2/FindReferences/FindReferencesTests.WinmdSymbols.vb

```diff
@@ -633,7 +633,171 @@ End Class
         </Document>
     </Project>
 </Workspace>
-            Await TestAPIAndFeature(input, kind, host)
+            Await TestAPI(input, host)
+        End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestWinmdVBAllIsWellTest_Feature(host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferencesWinRT="true">
+        <CompilationOptions OutputType="WindowsRuntimeMetadata"/>
+        <Document>
+Imports System
+Imports System.Collections
+Imports System.Collections.Generic
+Public Interface I
+    Sub Add(ByVal item As Integer)
+End Interface
+
+Public NotInheritable Class Class1
+    Implements I, IList(Of Integer)
```

Comment:
💭 Could we not use `ICollection(Of Integer)` instead of `IList(Of Integer)`?

Reply:
will check :)